### DISCUSSION TITLE
"You cannot delete your only Hobby team."

### DISCUSSION THREAD

**Discussion by stanlrt** [vercel/community#7293] [Help] [answered]
### Summary

Hello,

We had the default Hobby team. Then when connecting our GitHub repo, it asked us to create a new Pro team. We now want to delete the unused Hobby team, but it says "You cannot delete your only Hobby team.".

### Steps to Reproduce

1. Create a new Pro team
2. Attempt to delete old Hobby team

**Comment by versecafe** [comment, marked as answer]
@stanlrt You need at least one team to serve as your base Vercel account, the hobby team doesn't cost you anything and is meant for things like trying out new ideas, personal projects, etc.

**Reply by stanlrt** [reply]
Ok, makes sense

[a comment by an automated account is not shown]
